packages/react-reconciler/src/ReactFiberUnwindWork.js

```diff
@@ -211,8 +211,9 @@ function throwException(
     workInProgress = returnFiber;
     do {
       if (workInProgress.tag === SuspenseComponent) {
+        const fallback = workInProgress.memoizedProps.fallback;
         const didTimeout = workInProgress.memoizedState;
-        if (!didTimeout) {
+        if (fallback !== undefined && !didTimeout) {
```

Comment:
Also what happens if you sometimes specify it and sometimes don't? Is that a legit or broken pattern? Does it need a warning, like controlled<->uncontrolled "flapping" input?

Reply:
Prior art for our APIs (particularly defaultProps) is that undefined is treated as absent. Otherwise it is a pain to wrap Placeholder in your own component that passes a `fallback` through.